src/components/Warnings/AMPLWarning.tsx

```diff
@@ -0,0 +1,17 @@
+import { Trans } from '@lingui/macro';
+
+import { Link } from '../primitives/Link';
+
+export const AMPLWarning = () => {
+  return (
+    <Trans>
+      <b>Ampleforth</b> is a rebasing asset. Visit the{' '}
+      <Link href="https://docs.aave.com/developers/v/2.0/guides/ampl-asset-listing">
+        <Trans>
+          <u>documentation</u>
```

Comment:
`<u>` is unnecessary here.  You can use the `<Link underline="always" /> prop instead.  And we should try to avoid putting any HTML within the Trans tags